esmvaltool/recipes/recipe_gier2020bg.yml

```diff
@@ -330,8 +338,8 @@ diagnostics:
         var_plotnames: ["XCO~B~2~N~", "CO~B~2~N~"]
         # Overwrite station altitudes in case timeseries considered uses a
         # different altitude than the current one
-        overwrite_altitudes: [2710, 94, 248, 868, 482]
-        output_map: true
+        overwrite_altitudes: [2710, 94, 248, 482]  # LEF altitude: 868 to be inserted before 482
```

Comment:
is this mentioned/explained in the recipe documentation?

Reply:
Yep! Under user settings for that diagnostic